files/zh-cn/learn/tools_and_testing/github/index.md

```diff
@@ -38,25 +38,25 @@ slug: Learn/Tools_and_testing/GitHub
 
 先提醒一下这些链接将会带你去访问一些外部资源。我们最终将致力于开发我们专属的 Git 和 GitHub 教程，但现在，这些资料将会带你轻松入门。
 
-- [Hello World【欢迎来到新世界】 (来自 GitHub)](https://guides.github.com/activities/hello-world/)
+- [Hello World【欢迎来到新世界】 (来自 GitHub)](https://docs.github.com/zh/get-started/quickstart/hello-world)
   - : 这里是开始使用 GitHub 的正大门，在这里你可以学到 Git 的基础操作，例如：创建**储存库 (Repository)** 和**分支 (Brunch)** 、**确认提交 (commit)** 以及打开和合并**拉取请求** (open/merge **Pull Request**)。
-- [Git Handbook【使用手册】 (来自 GitHub)](https://guides.github.com/introduction/git-handbook/)
+- [Git Handbook【使用手册】 (来自 GitHub)](https://docs.github.com/zh/get-started/using-git/about-git)
```

Comment:
Hey @uiolee. Could you also update the link texts: just use the translated title without the original en-US title.

Reply:
ok